src/utils/rasterOptionFunctions.js

```diff
@@ -163,13 +176,21 @@ export function validateStyleObj(style) {
     parseFloat(style.max) &&
     parseFloat(style.min) > parseFloat(style.max)
   ) {
-    return false;
+    return {
+      validated:false,
+      errorMessage: "Minimum must be smaller than maximum."
+    };
   }
 
   if (style.colorMap && style.colorMap !== "") {
-    return true;
+    return {
+      validated:true,
+    };
   } else {
-    return false;
+    return {
+      validated:false,
+      errorMessage: "Please choose a color map."
+    };
   }
```

Comment:
Ik zie dat je hier validate functies hebt met een object, een boolean validated en eventueel een error message. De rest van de validation functies hadden tot nu alleen een string met een error message bij een fout, en 'false' als er geen validatiefout was (dat had ook wel iets anders kunnen zijn, maar ok).

Heb je het overal veranderd?

Reply:
Deze wordt dus alleen intern gebruikt en wordt aan de api overgezet naar het oorspronkelijke formaat. Toch willen we liever alles gelijk houden.
Dit ticket:
https://nelen-schuurmans.atlassian.net/browse/FRNT-321